### PR TITLE
Website: Initiative Page

### PR DESCRIPTION
**Pull request by cchrischen**
### Summary 
This PR implements the final page of the website -- the initiative page. Initiative information is stored in a JSON at `components/initiative/data`.  `eventName` is used to bold the corresponding names.

### Notion/Figma Link 
[Figma](https://www.figma.com/design/1nUpvsfgc9eftTyoHt7XuN/Website-Designs-SP24?node-id=1060-2264&node-type=canvas&t=VQg2ageMrVZCQSa0-0)
[Notion](https://www.notion.so/Initiatives-Page-1040ad723ce180c1a16ff36e85345b38)

### Test Plan 
Manual testing via resizing.

https://github.com/user-attachments/assets/8e1269fa-f194-40a0-9cd8-b27e7c7a53e3

### CODE REVIEW

File: new-dti-website/components/initiative/InitiativeDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  return (
+    <div className={`flex flex-col gap-10 ${className}`}>
+      <div className="flex flex-col gap-[14px]">
+        <div className="flex gap-2 min-h-10 xs:flex-col lg:flex-row">
+          <Image {...icon} />
+          <h3 className="font-semibold text-[32px] leading-10">{title}</h3>
+        </div>
+        <p className="font-semibold text-[22px] leading-[26px]">{subtitle}</p>
+        <div className="flex justify-center overflow-hidden rounded-md">
+          <Image {...image} className="h-[260px] max-w-none" />
```
Comment (JasonMun7) [inline new-dti-website/components/initiative/InitiativeDisplay.tsx:41]: From the Figma, the images should be a bit more rounded and have a drop shadow! I see that you used rounded-md but it doesn't show in the video you provided! Looking Great though!

Comment (JasonMun7) [reply new-dti-website/components/initiative/InitiativeDisplay.tsx:41]: Also, I don't know if this was intended but some of the images also appear to have a border in the figma. Might be good to ask Vannessa about this.

File: new-dti-website/src/app/initiative/page.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      <div className="max-w-7xl w-full flex flex-col lg:gap-6 xs:gap-3">
+        <h3 className="font-semibold md:text-[32px] xs:text-[22px] leading-10">
+          Want us to be a part of your next event?
+        </h3>
+        <p className="md:text-[22px] md:leading-[26px] xs:text-[12px] xs:leading-[14px]">
+          Feel free to coordinate with us over email, coffee, lunch-we're excited to work with you.
+        </p>
+        <button
+          className="rounded-xl py-3 px-[20px] bg-[#A52424] text-white 
+          font-bold hover:bg-[#D63D3D] w-fit"
```
Comment (JasonMun7) [inline new-dti-website/src/app/initiative/page.tsx:57]: I think maybe for the future we should think about adding the Design System Colors into the tailwind config. Can definitely do this later though.

File: new-dti-website/components/initiative/InitiativeDisplay.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+          width >= LAPTOP_BREAKPOINT ? inter.className : ibm_plex_mono.className
+        }`}
+      >
+        {parts.map((part, index) => (part === eventName ? <b key={index}>{part}</b> : part))}
+      </p>
+    </div>
+  );
+};
+
+const InitiativeDisplay = () => {
```
Comment (JasonMun7) [inline new-dti-website/components/initiative/InitiativeDisplay.tsx:55]: Great job on breaking these up into easier-to-read components!

File: new-dti-website/components/initiative/InitiativeDisplay.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
+import Image, { ImageProps } from 'next/image';
+import { Inter } from 'next/font/google';
+import initiatives from './data/initiatives.json';
+import useScreenSize from '../../src/hooks/useScreenSize';
+import { LAPTOP_BREAKPOINT } from '../../src/consts';
+import { ibm_plex_mono } from '../../src/app/layout';
+
+const inter = Inter({ subsets: ['latin'] });
```
Comment (JasonMun7) [inline new-dti-website/components/initiative/InitiativeDisplay.tsx:8]: Again, will probably be for the future when we get all the pages done, but we could also add this to the config for the future or the global css.

**Review by JasonMun7 — APPROVED** [review]
Hey Chris! Awesome job with the initiative page and for getting it done so quickly! I just left of couple of very small remarks. Overall great job on making it very readable! I was very easily able to scan through with no confusions!

File: new-dti-website/components/initiative/InitiativeDisplay.tsx (hunk 1 above)
Comment (andrew032011) [reply new-dti-website/components/initiative/InitiativeDisplay.tsx:41]: +1, I think the intention was to make these rounded

**Review by andrew032011 — APPROVED** [review]
Looks good after addressing Jason's comments!

**Comment by andrew032011** [comment]
Screenshot 2024-10-29 at 12 01 06 AM
The images look like they're not fitting into the frames.

**Comment by andrew032011** [comment]
looks good for computer view! can we also align it on the left for mobile view?
Screenshot 2024-11-03 at 3 12 25 PM

**Review by andrew032011 — APPROVED** [review]

**Pull request merged**